### PR TITLE
Updated scripts to use python3

### PR DESCRIPTION
**Pull request by jamesfwood**
Updates scripts to python3.  Make sure to update your locale to support UTC-8.

### CODE REVIEW

File: mrf_apps/mrf_read.py [hunk 1]
```diff
@@ -107,7 +107,7 @@
 
     if len(rasterElem.getElementsByTagName('Size')):
         sizeElem  = rasterElem.getElementsByTagName('Size')[0]
-        sizeAttrs = dict(sizeElem.attributes.items())
+        sizeAttrs = dict(list(sizeElem.attributes.items()))
```
Comment (lucianpls) [inline mrf_apps/mrf_read.py:110]: this seems to say:
sizeAttrs = sizeElem.attributes
Why take a dictionary, convert it to list and then back to a dictionary?  A deepcopy doesn't seem to be needed.

**Review by lucianpls — CHANGES_REQUESTED** [review]

File: mrf_apps/mrf_read.py (hunk 1 above)
Comment (jamesfwood) [reply mrf_apps/mrf_read.py:110]: I ran 2to3 on the script and this is something it auto corrected.  Should I put it back to original or leave it like this?

File: mrf_apps/mrf_join.py [hunk 2]
```diff
@@ -77,7 +78,7 @@ def mrf_join(argv, forceoffset = None):
 
     # At this point the output exist, loop over the inputs
     for input_file in argv[:-1]:
-        print("Processing {}".format(input_file))
+        print(("Processing {}".format(input_file)))
```
[the remark this answers is not in the document]
Comment (jamesfwood) [reply mrf_apps/mrf_join.py]: Reverted back

File: mrf_apps/mrf_join.py [hunk 3]
```diff
@@ -144,11 +145,11 @@ def getmrfinfo(fname):
     assert root.tag == "MRF_META", "{} is not an MRF".format(fname)
     info = {}
     info['size'] = { key : int(val) for (key, val) in 
-                    root.find("./Raster/Size").attrib.items() }
+                    list(root.find("./Raster/Size").attrib.items()) }
```
[the remark this answers is not in the document]
Comment (jamesfwood) [reply mrf_apps/mrf_join.py]: Reverted back

File: mrf_apps/mrf_join.py [hunk 4]
```diff
@@ -144,11 +145,11 @@ def getmrfinfo(fname):
...
 
     if root.find("./Raster/PageSize"):
         info['pagesize'] = { key : int(val) for (key, val) in 
-                        root.find("./Raster/PageSize").attrib.items() }
+                        list(root.find("./Raster/PageSize").attrib.items()) }
```
[the remark this answers is not in the document]
Comment (jamesfwood) [reply mrf_apps/mrf_join.py]: Reverted back

File: mrf_apps/mrf_read.py (hunk 1 above)
Comment (lucianpls) [reply mrf_apps/mrf_read.py:110]: I think the original was fine, but test.
Don't know why 2to3 insists on converting things to list, looks like "better safe than sorry" principle.

**Review by jtroberts — APPROVED** [review]

File: mrf_apps/mrf_read.py (hunk 1 above)
Comment (jamesfwood) [reply mrf_apps/mrf_read.py]: I reverted it back to original

**Comment by jamesfwood** [comment]
It should be ready for merge now.  Thanks!

**Review by lucianpls — APPROVED** [review]

**Pull request merged**